components/chart/plot-bands.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Plot Bands
+page_title: Plot Bands
+description: Plot Bands in the Charts for Blazor.
+slug: chart-plot-bands
+tags: telerik,blazor,chart,plot,bands,plotbands
+published: true
+position: 45
+---
+
+# Plot Bands
+
+Plot Bands are colored ranges in the Telerik UI for Blazor Chart, the purpose of which is to highlight areas of the chart by changing its background in a predefined axis range.
+
+The plot bands appear on the chart as colored areas that fill the entire space between two grid lines on the x-axis or on the y-axis.
+
+
+## Creating Plot Bands
+
+1. Add instances of the `PlotBand` items to the `PlotBands` collection of the X or Y axes. 
+1. Set a value to the `From` and `To` parameters.
+1. Provide a valid CSS color to the `Color` parameter.
+1. (Optional) Customize the plot band further by providing values to the other exposed parameters.
```

Comment:
This step does not provide specific value here, because we don't mention what are these other parameters. We can move the sentence somewhere else and provide more details.